features/configuration_files/show_configuration_file.feature

```diff
@@ -0,0 +1,46 @@
+Feature: Show configuration file
+
+  Scenario: No configuration file present
+    Given the clean file 'clean.rb'
+    When there is no configuration file and I run reek --show-configuration-path clean.rb
```

Comment:
@mvz i just cant get this one scenario to pass because it will always find the .reek.yml in our root directory. The corresponding "allow" in the step definition seems to have no effect. Any idea how to fix that? If we cant fix it i see the following options:

* Drop the scenario. Not great.
* Test that scenario via rspec. Also not great because this will be super confusing
* Extend application_spec or use sth like rspec_command and move the whole feature there. This could also be the first step of getting rid of cucumber in our test suite

Reply:
Here are my thoughts:

* The reason this doesn't work is that Cucumber runs the actual Reek script, which won't know anything about any `allow` called in the steps.
* We should definitely test this, either in Cucumber or RSpec.
* I don't think `rspec_command` will buy us anything, since it will have the same problem (but see below).
* If we run the scenarios in some temp dir outside of regular `$HOME`, we can really simulate the case where no file is found. I *think* Aruba lets you do this, but I'm not sure. If `rspec_command` does this out of the box, that would be a reason to switch.
* In general I prefer to just run the 'happy path' in a cucumber scenario, and test the edge cases in RSpec. This would avoid the entire issue, since we can just test this the same way we test `ConfigurationFileFinder`, and we could have just one scenario showing the output in the 'normal' case of a project-specific file.

So, my recommendation is: Use just one scenario to exercise the feature, and test the rest in RSpec.